client/.env.local.sample

```diff
@@ -1,3 +1,18 @@
+# =================================
+# =================================
+# CLIENT-SIDE ENVIRONMENT VARIABLES
+# =================================
+# =================================
+#
+# To see documentation on server-side variables, 
+# head over to the `.env.sample` file in the parent directory
+#
+# Note that some client-side variables are committed to git 
+# via the `.env` file in this directory. This follows the protocol
+# for the "Create React App" framework and allows certain public
+# variables to get used in production more easily. 
```

Comment:
Again, not sure if this is the clearest explanation of _why_ we have this extra client-side env file.